srcpkgs/recutils/template

```diff
@@ -1,17 +1,25 @@
 # Template file for 'recutils'
 pkgname=recutils
 version=1.8
-revision=1
+revision=2
 build_style=gnu-configure
 configure_args="--with-bash-headers --disable-rpath"
-hostmakedepends="pkg-config"
+# TODO remove automake in 1.9; only needed for copy-file
+hostmakedepends="automake pkg-config"
 makedepends="acl-devel libgcrypt-devel libuuid-devel libcurl-devel"
+pre_configure() {
+	# XXX horrible kludge to avoid help2man - remove in 1.9
+	touch man/*.1
+	# XXX ONLY needed for copy-file - remove in 1.9
+	aclocal && automake
+}
```

Comment:
This should go below the information block, above `librec1_package`librec1_package

Reply:
Done and added the version check.

Reply:
About the size of the patch, I was thinking that you could, more or less, clone the gnulib repo, then check for every file that is quoted in the patch that it's there semi-automatically, something like
```
$ fgrep -C3 "$(cat somefile.c | sed 's/^/+/g)" huge.patch
```

The "-C3" is to see that the patch begins and ends with the files; what is left is a few lines and most of them are trivial,
like adding the new files to the makefile.